.github/ISSUE_TEMPLATE/data_bug_report.md

```diff
@@ -0,0 +1,31 @@
+---
+name: Report a bug - data
+about: Report a bug with your data build
+title: ''
+labels: 'bug'
+assignees: ''
+
+---
+**Please complete the following information**
+Data repository URL: [e.g. github.com/ONSdigital/sdg-indicators]
+Data or metadata issue:
```

Comment:
Is "Data or metadata issue" necessary? How is it different from what they would write under "Describe the bug"?

Reply:
Oh! I see, this is meant to ask whether the problem is related to data or metadata. Maybe this could be clarified, or possibly removed if we think it might be covered in the bug description.